src/services/iasql.ts

```diff
@@ -127,9 +130,10 @@ export async function remove(dbAlias: string, user: any) {
   let conn;
   try {
     const { dbId, dbUser } = await dbMan.getMetadata(dbAlias, user);
+    scheduler.stop(dbId);
     conn = await createConnection(dbMan.baseConnConfig);
     await conn.query(`
-      DROP DATABASE ${dbId} WITH (FORCE);
+      DROP DATABASE ${dbId};
```

Comment:
this was causing connections to be closed forcefully which made the scheduler unhappy. Here we wait for clean up after itself gracefully.